package.json

```diff
@@ -7,7 +7,7 @@
     "extends @edx/browserslist-config"
   ],
   "dependencies": {
-    "@edx/brand": "npm:@edx/brand-openedx@1.2.0",
+    "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
```

Comment:
nit: We generally pin dependencies in `package.json` (without the `^`).

Reply:
got it, I'll update.